doc/developer-notes.md

```diff
@@ -595,10 +595,9 @@ Common misconceptions are clarified in those sections:
 
   - *Rationale*: This avoids memory and resource leaks, and ensures exception safety.
 
-- Use `MakeUnique()` to construct objects owned by `unique_ptr`s.
+- Use `std::make_unique` to construct objects owned by `unique_ptr`s.
```

Comment:
I don't think this needs to go in our developer notes at all. It's general best practice when writing modern c++ that you should use std::make_unique to construct unique pointers. We don't need to document all those best practices in our project.

Reply:
This has now been removed.